docs/source/upgrade/ldap-migration.md

```diff
@@ -0,0 +1,241 @@
+# Overview
+
+This documentation demonstrates how to upgrade from CN 4.2 LDAP to 4.5 PostgreSQL.
```

Comment:
We can say >= 4.2 LDAP to 4.5 postgres. Apply this notion across the doc because this will work for versions 4.3 as well.